mongomock/aggregate.py

```diff
@@ -358,6 +358,37 @@ def _handle_date_operator(self, operator, values):
             return out_value.second
         if operator == '$millisecond':
             return int(out_value.microsecond / 1000)
+        if operator == '$dateToString':
+            if not isinstance(values, dict):
+                raise OperationFailure(
+                    '$dateToString operator must correspond a dict'
+                    'that has "format" and "date" field.'
+                )
+            if 'format' not in values.keys() or 'date' not in values.keys():
+                raise OperationFailure(
+                    '$dateToString operator must correspond a dict'
+                    'that has "format" and "date" field.'
+                )
+            if '%L' in out_value['format']:
+                raise NotImplementedError(
+                    'Although %L is a valid date format for the '
+                    '$dateToString operator, it is currently not implemented '
+                    ' in Mongomock.'
+                )
+            if 'onNull' in values.keys():
```

Comment:
No need to check `.keys()`, just `if 'onNull' in values:`

Reply:
Done